app/src/androidTest/java/net/skyscanner/backpack/compose/dialog/BpkDialogTest.kt

```diff
@@ -148,22 +145,24 @@ class BpkDialogTest : BpkSnapshotTest() {
 
     val view = composeTestRule.onNode(isDialog()).fetchRootView()
 
-    rule.scenario.onActivity { activity ->
+    rule.scenario.waitForActivity().also { activity ->
```

Comment:
Could you link to the issue?

Reply:
Checked the issue tracker and there's none - we could create one, but I suspect it's a system limitation for some reason otherwise they would've provided support (this is not a shot issue, but of the android snapshot functionality)